exercises/protein-translation/canonical-data.json

```
@@ -242,6 +242,15 @@
       },
       "expected": ["Tryptophan", "Cysteine", "Tyrosine"]
     },
+    {
+      "uuid": "f6f92714-769f-4187-9524-e353e8a41a80",
+      "description": "Sequence of two non-STOP codons is not matched to a STOP codon",
```

```suggestion
      "description": "Sequence of two non-STOP codons does not translate to a STOP codon",
```
Other test descriptions reference translating, not matching.